lib/Runtime/ByteCode/ByteCodeGenerator.cpp

```diff
@@ -2075,7 +2075,9 @@ void ByteCodeGenerator::CheckDeferParseHasMaybeEscapedNestedFunc()
     else
     {
         // We have to wait until it is parsed before we populate the stack nested func parent.
-        FuncInfo * parentFunc = top->GetParamScope() ? top->GetParamScope()->GetEnclosingFunc() : top->GetBodyScope()->GetEnclosingFunc();
+        FuncInfo * parentFunc = top->GetParamScope() ? top->GetParamScope()->GetEnclosingFunc() :
+                                top->GetBodyScope() ? top->GetBodyScope()->GetEnclosingFunc() :
+                                top->GetFuncExprScope()->GetEnclosingFunc();
```

Comment:
>GetFuncExprScope [](start = 37, length = 16)

Could GetFuncExprScope return null?

Reply:
I don't believe it's possible for all of these to be null.